src/app/pages/categories/management/pages-transaction-categories.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { map, Observable, Subject } from 'rxjs';
+import { DomainsTransactionCategoriesGateway } from 'src/app/domains/categories/domains.transaction-categories.gateway';
+import { TransactionCategory } from '../transaction-category.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PagesTransactionCategoriesService {
```

Comment:
I would move it one level up at least to `/pages/categories` considering it will be used in wallet editor

Reply:
Was not done yet